templates/customer/index.html.twig

```diff
@@ -0,0 +1,57 @@
+{% use "bootstrap_4_layout.html.twig" %}
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %}Money Management v0.01a - Cadastro de Cliente{% endblock %}</title>
+    {% block stylesheets %}<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">{% endblock %}
+</head>
+<body>
+
+{% block body %}
+
+    {% block header %}
+        {{ parent() }}
+    {% endblock header %}
+
+    <div class="container">
+        <h1 class="text-center">Clientes</h1>
+        <table class="table">
+            <thead>
+                <tr>
+                    <th scope="col">Cod</th>
+                    <th scope="col">Nome</th>
+                    <th scope="col">Doc.Numero</th>
+                    <th scope="col">Email</th>
+                    <th scope="col">Telefone</th>
+                    <th scope="col">Ação</th>
+                </tr>
+            </thead>
+            <tbody>
+                 {% for customer in customers %}
+                    <tr>
+                        <td>{{customer.id}}</td>
+                        <td>{{customer.name}}</td>
+                        <td>{{customer.documentnumber}}</td>
+                        <td>{{customer.email}}</td>
+                        <td>{{customer.phone}}</td>
+                        <td>
+                            <a class="btn btn-sm btn-info" href="/edit_customer/{{customer.id}}">Editar</a>
```

Comment:
What about changing the button style to the `btn-secondary` option? I think that gray colour is a lot more neutral than the cyan one.

Reply:
Ok, i'm following your model in branch crud users